src/sign.ts

```diff
@@ -32,22 +32,15 @@ async function sign(): Promise<void> {
         // inputs from user
         const key_id = core.getInput('key_id');
         const plugin_config = core.getInput('plugin_config');
+        const pluginConfigList: string[] = getPluginConfigList(plugin_config);
```

Comment:
nit: do we need to explicitly define `string[]` for `pluginConfigList`? Is it auto determined by typescript?

Reply:
yeah, it can be recognized by typescript, removing it.